File: cs_t4_full/Clone_908.cs
public static long fibo_n (long N) {
    if (N <= 0)
        return 0;
    if (N == 1)
        return 1;
    if (N <= 4)
        return N - 1;
    return fibo_n (N - 1) + fibo_n (N - 2);
}


 public static int fibo_n (int number) {
    if (number <= 0) 
        return 0;
    if (number == 1 || number == 2) 
        return 1;
    if (number == 3)
        return 2;
    if (number == 4) 
        return 3;
    return fibo_n (number - 1) + fibo_n (number - 2);
}

File: c_t4_full/Clone_908.c
int main (int argc, char **argv) {
    int val = -1;
    char other;
    int code;
    if (argc != 2) {
        printf ("please give an arg to program\n");
        exit (0);
    }
    code = sscanf (argv[1], "%d", &val);
    if (code == 1) {
        printf ("number %d", val);
    }
    else {
        code = sscanf (argv[1], "%c", &other);
        if (code == 1) {
            printf ("character is %c", other);
        }
        else {
            printf ("error occured");
        }
    }
}





int main(int argc, char** argv) {
	    if (argc != 2) {
		printf("please give an arg to program\n");
		exit(0);
	    }
	    char first = argv[1][0];
	    if (isdigit(first)) {
		int val = atoi(argv[1]);
		printf("number %d", val);
	    } else {
		printf("character is %c", first);
	    }
}


